distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/AbstractIntegrationTest.java

```diff
@@ -166,11 +172,33 @@ public void initFacades() {
      */
     @org.ops4j.pax.exam.Configuration
     public Option[] config() throws URISyntaxException {
+        basePort = findPortNumber(20000);
+
+        HTTP_PORT = String.valueOf(basePort + 1);
```

Comment:
Could also create a static inner class (e.g., `Ports`) that calls `findPortNumber` in its ctor and has getters for the different port values.